sigle/src/modules/editor/TipTapEditor.tsx

```diff
@@ -173,7 +177,7 @@ export const TipTapEditor = forwardRef<
       TipTapPlaceholder,
       // Custom extensions
       SlashCommands.configure({
-        commands: slashCommands({ storyId: story.id }),
+        commands: !isMobile ? slashCommands({ storyId: story.id }) : [],
```

Comment:
here the slash command plugin is still injected in the plugin list, removing it totally is better so no code is executed :)

Reply:
do you know what is the best method to only prevent it from being injected on mobile? Typescript is throwing an error if I use a conditional/`&&` operator as boolean is incompatible with the type the extension is expecting. Is there another way we can do it?

Reply:
Would smth like this work?
```
    isMobile
        ? SlashCommands.configure({
            commands: slashCommands({ storyId: story.id }),
          })
        : undefined,
    ].filter((pluging) => !!plugin),
```

Reply:
Typescript is still throwing an error with this regarding undefined not being assignable :/